integration-test.py

```diff
@@ -159,21 +159,57 @@ def match_distance(actual, expected):
     return (distance, misses)
 
 
-@contextmanager
-def features_in_tile_layer(z, x, y, layer):
+class GatewayTimeout(Exception):
+    def __init__(self, message):
+        self.message = message
```

Comment:
Yea, exactly. `pass` is just like a "nothing more to see here" placeholder. You can alternatively add something like a docstring and it'll work the same way. For example:

```python
class GatewayTimeout(Exception):
    """Raised when requests for tiles time out"""
```